src/modal.js

```diff
@@ -47,11 +47,13 @@ const ModalWrapper = p => {
   useEffect(() => {
     const self = modal.current
     if (ready || !self) return
+    const subscribed = true;
```

Comment:
```suggestion
    let subscribed = true
```
Think this needs to be mutable so it can be updated in the `finally`